firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/type/Part.kt

```diff
@@ -93,7 +95,11 @@ public class FunctionResponsePart(public val name: String, public val response:
   @Serializable
   internal data class Internal(val functionResponse: FunctionResponse) : InternalPart {
 
-    @Serializable internal data class FunctionResponse(val name: String, val response: JsonObject)
+    @Serializable public data class FunctionResponse(val name: String, val response: JsonObject)
```

Comment:
A public inner class inside an internal class is a really weird pattern, I'm not sure if you can even refer to it. We should have a top level `FunctionResponse` API class and a `FunctionResponse.Internal` Serializable class inside it

Reply:
ohh sorry i forgot to change it back to internal.. it doesnt have to be public.